workspace.sh

```diff
@@ -206,7 +207,9 @@ workspace() {
         ;;
     "in")
         shift
+        [ -z "${1:-}" ] && { show_help; return 1; }
         WORKSPACE="$1"; shift
+        [ -z "$@" ] && { show_help; return 1; }
```

Comment:
Expanding `"$@"` in `[ ... ]` can lead to arguments being interpreted by `test`. Use `$#` for checking the number of arguments.